src/middlewares/userMiddlewares.js

```diff
@@ -7,27 +9,71 @@ import statusCode from '../utils/statusCode';
 const { errorResponse } = response;
 const { findUserByEmail } = authServices;
 
-export default {
-	confirmUserEmail: async (req, res, next) => {
-		const { email } = req.userData;
-		if (!email || !email.trim()) {
-			return errorResponse(res, statusCode.badRequest, message.emptyEmail);
-		}
+export const confirmUserEmail = async (req, res, next) => {
+	const { email } = req.userData;
+	if (!email || !email.trim()) {
+		return errorResponse(res, statusCode.badRequest, message.emptyEmail);
+	}
+
+	const user = await findUserByEmail(email);
+	if (!user) {
+		return errorResponse(
+			res,
+			statusCode.notFound,
+			message.unregisteredEmail(email)
+		);
+	}
+
+	req.body = {
+		...req.body,
+		full_name: `${user.first_name} ${user.last_name}`,
+		email,
+	};
+	return next();
+};
 
-		const user = await findUserByEmail(email);
-		if (!user) {
+export const verifyTravelOwner = async (req, res, next) => {
+	try {
+		const { travel_id } = req.params;
+		const travelObj = await travelServices.findTravelById(travel_id);
+
+		if (travelObj === null) {
+			return errorResponse(res, statusCode.notFound, message.travelNotFound);
+		}
+		const travel = travelObj.dataValues;
+		if (parseInt(travel.user_id, 10) !== req.userData.id
+			/* && req.userData.id !== dept.line_manager_id */) {
 			return errorResponse(
 				res,
-				statusCode.notFound,
-				message.unregisteredEmail(email)
+				statusCode.unauthorized,
+				message.unauthorizedAccessToTravel
 			);
 		}
+		return next();
+	} catch (err) {
+		errorResponse(res, statusCode.serverError, err);
+	}
+};
 
-		req.body = {
-			...req.body,
-			full_name: `${user.first_name} ${user.last_name}`,
-			email,
-		};
+export const verifyCommentOwner = async (req, res, next) => {
+	try {
+		const { comment_id } = req.params;
```

Comment:
It would make sense to reduce lines of code if you are not using {comment_id} as a variable anywhere.